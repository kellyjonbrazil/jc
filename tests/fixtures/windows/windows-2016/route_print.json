{
  "interface_list": [
    {
      "interface_index": 2,
      "mac_address": "00:0c:29:a1:38:2c",
      "description": "Intel(R) 82574L Gigabit Network Connection"
    },
    {
      "interface_index": 1,
      "mac_address": null,
      "description": "Software Loopback Interface 1"
    },
    {
      "interface_index": 6,
      "mac_address": null,
      "description": "Microsoft ISATAP Adapter #2"
    }
  ],
  "ipv4_route_table": {
    "active_routes": [
      {
        "network_destination": "0.0.0.0",
        "netmask": "0.0.0.0",
        "gateway": "192.168.22.1",
        "interface": "192.168.22.153",
        "metric": "281"
      },
      {
        "network_destination": "127.0.0.0",
        "netmask": "255.0.0.0",
        "gateway": "On-link",
        "interface": "127.0.0.1",
        "metric": "331"
      },
      {
        "network_destination": "127.0.0.1",
        "netmask": "255.255.255.255",
        "gateway": "On-link",
        "interface": "127.0.0.1",
        "metric": "331"
      },
      {
        "network_destination": "127.255.255.255",
        "netmask": "255.255.255.255",
        "gateway": "On-link",
        "interface": "127.0.0.1",
        "metric": "331"
      },
      {
        "network_destination": "192.168.22.0",
        "netmask": "255.255.255.0",
        "gateway": "On-link",
        "interface": "192.168.22.153",
        "metric": "281"
      },
      {
        "network_destination": "192.168.22.153",
        "netmask": "255.255.255.255",
        "gateway": "On-link",
        "interface": "192.168.22.153",
        "metric": "281"
      },
      {
        "network_destination": "192.168.22.255",
        "netmask": "255.255.255.255",
        "gateway": "On-link",
        "interface": "192.168.22.153",
        "metric": "281"
      },
      {
        "network_destination": "224.0.0.0",
        "netmask": "240.0.0.0",
        "gateway": "On-link",
        "interface": "127.0.0.1",
        "metric": "331"
      },
      {
        "network_destination": "224.0.0.0",
        "netmask": "240.0.0.0",
        "gateway": "On-link",
        "interface": "192.168.22.153",
        "metric": "281"
      },
      {
        "network_destination": "255.255.255.255",
        "netmask": "255.255.255.255",
        "gateway": "On-link",
        "interface": "127.0.0.1",
        "metric": "331"
      },
      {
        "network_destination": "255.255.255.255",
        "netmask": "255.255.255.255",
        "gateway": "On-link",
        "interface": "192.168.22.153",
        "metric": "281"
      }
    ],
    "persistent_routes": [
      {
        "network_address": "0.0.0.0",
        "netmask": "0.0.0.0",
        "gateway_address": "192.168.22.1",
        "metric": "Default"
      }
    ]
  },
  "ipv6_route_table": {
    "active_routes": [
      {
        "interface": 1,
        "metric": "331",
        "network_destination": "::1/128",
        "gateway": "On-link"
      },
      {
        "interface": 2,
        "metric": "281",
        "network_destination": "fe80::/64",
        "gateway": "On-link"
      },
      {
        "interface": 2,
        "metric": "281",
        "network_destination": "fe80::40c:2f5f:6658:9dc3/128",
        "gateway": "On-link"
      },
      {
        "interface": 1,
        "metric": "331",
        "network_destination": "ff00::/8",
        "gateway": "On-link"
      },
      {
        "interface": 2,
        "metric": "281",
        "network_destination": "ff00::/8",
        "gateway": "On-link"
      }
    ],
    "persistent_routes": []
  }
}
