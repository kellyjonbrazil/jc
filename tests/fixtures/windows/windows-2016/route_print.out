===========================================================================
Interface List
  2...00 0c 29 a1 38 2c ......Intel(R) 82574L Gigabit Network Connection
  1...........................Software Loopback Interface 1
  6...00 00 00 00 00 00 00 e0 Microsoft ISATAP Adapter #2
===========================================================================

IPv4 Route Table
===========================================================================
Active Routes:
Network Destination        Netmask          Gateway       Interface  Metric
          0.0.0.0          0.0.0.0     192.168.22.1   192.168.22.153    281
        127.0.0.0        255.0.0.0         On-link         127.0.0.1    331
        127.0.0.1  255.255.255.255         On-link         127.0.0.1    331
  127.255.255.255  255.255.255.255         On-link         127.0.0.1    331
     192.168.22.0    255.255.255.0         On-link    192.168.22.153    281
   192.168.22.153  255.255.255.255         On-link    192.168.22.153    281
   192.168.22.255  255.255.255.255         On-link    192.168.22.153    281
        224.0.0.0        240.0.0.0         On-link         127.0.0.1    331
        224.0.0.0        240.0.0.0         On-link    192.168.22.153    281
  255.255.255.255  255.255.255.255         On-link         127.0.0.1    331
  255.255.255.255  255.255.255.255         On-link    192.168.22.153    281
===========================================================================
Persistent Routes:
  Network Address          Netmask  Gateway Address  Metric
          0.0.0.0          0.0.0.0     192.168.22.1  Default
===========================================================================

IPv6 Route Table
===========================================================================
Active Routes:
 If Metric Network Destination      Gateway
  1    331 ::1/128                  On-link
  2    281 fe80::/64                On-link
  2    281 fe80::40c:2f5f:6658:9dc3/128
                                    On-link
  1    331 ff00::/8                 On-link
  2    281 ff00::/8                 On-link
===========================================================================
Persistent Routes:
  None